themis/languages/wasm/installation.md

```diff
@@ -78,49 +78,50 @@ WasmThemis uses [BoringSSL](https://boringssl.googlesource.com/boringssl/) as it
 BoringSSL is included as a submodule in the Themis repository,
 but you will need to install its build dependencies as well.
 
- 1. Install Emscripten toolchain
+ 1. Checkout the themis repository.
```

Comment:
```suggestion
 1. Checkout Themis repository.
```

Reply:
I'm not a native speaker but the “the *Proper-Noun* *of-common-group-noun-kind*” pattern seems to be quite popular. So I'd be happy to just capitalize Themis here.